pkg/kube/inject/webhook.go

```diff
@@ -588,6 +601,23 @@ func createPatch(pod *corev1.Pod, prevStatus *SidecarInjectionStatus, annotation
 	return json.Marshal(patch)
 }
 
+func extractCanonicalSerivceLabel(podLabels map[string]string, workloadName string) string {
```

Comment:
nit: Since this is the fallback method, It would be good to document it with the method signature.